### PR TITLE
include invitation code when creating a lobby - closes #159

### LINKED ISSUE #159: Modify the lobby class to create a code when a lobby is created.

### PR DESCRIPTION
**Pull request by luis-tm**
include invitation code when creating a lobby:
- length = 10
- alphanumeric

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java [hunk 1]
```diff
@@ -95,6 +99,22 @@ public void setGameType(GameType gameType) {
...
+    private String generateInvitationCode() {
+        //set limits for including only alphanumeric values
+        int lowerLimit = 48;
+        int upperLimit = 123;
+
+        //set limit for the string length
+        int lengthLimit = 10;
+
+        Random random = new Random();
+        return random.ints(lowerLimit, upperLimit)
```
Comment (apple00juice) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java:111]: I don't understand what you are completely doing, so maybe it is obvious, how do you check that the invitation code is unique over all lobbies ?

Comment (luis-tm) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java:111]: When joining the lobby, we are sending the id of the lobby (unique), and the code, which is going to be matched with the one stored in the DB. So I think that the code does not need to be unique.

Comment (apple00juice) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java:111]: Ahh, okey I interpreted the code as an additional way to identify a lobby, and you more like a password if I understand correctly? 
This naturally also works!

**Review by apple00juice — APPROVED** [review]

File: src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java (hunk 1 above)
Comment (apple00juice) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java:111]: nono, that's probably better.  Two different ways to identify a lobby are enough 😅

**Review by hilalcomak — APPROVED** [review]

**Pull request merged**